### DISCUSSION TITLE
upgrade to an actual openwrt and cve secure version

### DISCUSSION THREAD

**Discussion by heini66** [Ysurac/openmptcprouter#2762] [Ideas]
hi,
first of all, thxs for all the work and knowledge.
but:
openwrt 19 and all the belonging packages in it is ( easy said ) little outdated. any plans to move this to an actual  ( and https://www.cvedetails.com/vulnerability-list/ ) supporting  openwrt version?
many people would like to use this if it's a little more secure. ( my 2 cent's )

**Comment by Ysurac** [comment]
Hi,

OpenMPTCProuter kernel 5.4 is based on OpenWRT 21.02 (last release to support 5.4 kernel), 5.15 and 6.1 releases are based on master openwrt branch.

**Comment by heini66** [comment]
sorry for the rumor. then i've to poke thrugh my setup and dig why i'm on openwrt 19 with openmptc
